tfx/orchestration/airflow/airflow_dag_runner.py

```diff
@@ -111,3 +114,26 @@ def run(self, tfx_pipeline: pipeline.Pipeline):
             component_impl_map[upstream_node])
 
     return airflow_dag
+
+
+  def _replace_runtime_params(self, comp):
+    for k, prop in comp.spec.exec_properties.copy().items():
+      if isinstance(prop, RuntimeParameter):
+        
+        # Airflow only supports string parameters
+        if prop.ptype != str:
+          raise RuntimeError(
+            f'RuntimeParameter in Airflow does not support {prop.ptype} only ptype supported is string'
+          )
+        
+        # If default is a template, we can just remove the template markers to make it work.
+        if prop.default.startswith('{{') and prop.default.endswith('}}'):
+          default = prop.default.replace('{{','').replace('}}', '')
+        else:
+          default = f'"{prop.default}"'
```

Comment:
I think this might not properly quote strings with double-quotes already in them.  Can you use `default = repr(prop.default)` instead?

Reply:
Decided to use `json.dumps` since that uses `"` instead of `'` like repr. I believe this may make it also available to add more objects in default value as TFX will try to deserialize from json as well? Happy to use repr if deemed better.